addons/mail/static/src/components/activity_box/activity_box.xml

```diff
@@ -4,7 +4,7 @@
     <t t-name="mail.ActivityBox" owl="1">
         <div class="o_ActivityBox">
             <t t-if="chatter and chatter.thread">
-                <a role="button" class="o_ActivityBox_title btn" t-on-click="_onClickTitle">
+                <a href="javascript:;" role="button" class="o_ActivityBox_title btn" t-on-click="_onClickTitle" t-att-aria-expanded="chatter.isActivityBoxVisible ? 'true' : 'false'">
```

Comment:
href="#" and prevent default in handler.
Or is it possible to make it a div, and set tabindex and whatnot instead? It's definitely not a link...

Reply:
Thanks, I set a href.

> Or is it possible to make it a div, and set tabindex and whatnot instead? It's definitely not a link...

If an element without native keyboard support like div is used, that would require taking care of handling keyboard support manually, which is more complex than using a button or a link with button role.

I agree it's not a link, but link with button role is much more accurate than a div, span or similar, IMHO. Several current accessibility issues are currently due to elements that should be buttons but they are not, and they don't behave like so when interacting with the keyboard.